src/directives/map.js

```diff
@@ -44,6 +55,49 @@ ngeo.mapDirective = function(ngeoSyncDataSourcesMap) {
       map.setTarget(element[0]);
 
       ngeoSyncDataSourcesMap.map = map;
+
+
+      // Get the 'window resize' attributes, which are optionnal. If defined,
+      // the browser window 'resize' event is listened to update the size of
+      // the map when fired. A transition option is also available to let any
+      // animation that may occur on the div of the map to smootly resize the
+      // map while in progress.
+      const manageResizeAttr = 'ngeoMapManageResize';
+      const manageResizeProp = attrs[manageResizeAttr];
+      const manageResize = scope.$eval(manageResizeProp);
+
+      if (manageResize) {
+        const resizeTransitionAttr = 'ngeoMapResizeTransition';
+        const resizeTransitionProp = attrs[resizeTransitionAttr];
+        const resizeTransition = scope.$eval(resizeTransitionProp);
```

Comment:
Would it be okay if I did this?

`const resizeTransition = /**  {number|undefined} */ (scope.$eval(resizeTransitionProp));`